src/vabackend.c

```diff
@@ -675,19 +823,61 @@ static VAStatus nvQueryConfigAttributes(
     *profile = cfg->profile;
     *entrypoint = cfg->entrypoint;
     int i = 0;
+    attrib_list[i].value = 0;
```

Comment:
I believe NVDEC can decode all videos to NV12, so it should remain as the base RT.

Reply:
I'm not actually certain that 444 videos can be down sampled to NV12. It is possible to use the cuvidGetDecoderCaps API call to see what it's capable (my card doesn't support any 444 formats, so I can't check).

Reply:
According to https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Video_codecs#hevc_h.265 HEVC/VP9 profiles are backward compatible excepting VP9 profile 2/3 that doesn't support 8 bits (only 10 to 12).